### PR TITLE
[Admin] Receipt Reports

### PR DESCRIPTION
**Pull request by sampoder**
Adds a CSV exports to `/admin/users` which includes the total missing receipts. I need some sanity checks on the changes to `transactions_missing_receipts`. Closes https://github.com/hackclub/hcb/issues/5931/

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -246,8 +254,12 @@ def receipt_bin
 
   def transactions_missing_receipt
     @transactions_missing_receipt ||= begin
+      return [] unless stripe_cards.any? || emburse_cards.any?
+
       user_cards = stripe_cards.includes(:event).where.not(event: { category: :salary }) + emburse_cards.includes(:emburse_transactions)
```
Comment (garyhtou) [inline app/models/user.rb]:
```suggestion
      user_cards = stripe_cards.includes(:event).where.not(event: { category: :salary }) + emburse_cards.includes(:emburse_transactions)
      return [] unless user_cards.any?
```

**Review by garyhtou — COMMENTED** [review]
🪄

File: app/models/user.rb [hunk 2]
```diff
@@ -255,10 +255,10 @@ def receipt_bin
   def transactions_missing_receipt
     @transactions_missing_receipt ||= begin
       user_cards = stripe_cards.includes(:event).where.not(event: { category: :salary }) + emburse_cards.includes(:emburse_transactions)
-      return [] unless user_cards.any?
+      return HcbCode.none unless user_cards.any?
```
Comment (garyhtou) [inline app/models/user.rb]: ah good catch

Comment (sampoder) [reply app/models/user.rb]: yep!

**Comment by sampoder** [comment]
@RhysMP let me know when you want to use this!

**Pull request merged**